model/project_parser.go

```diff
@@ -1177,7 +1179,13 @@ func evaluateBuildVariants(tse *taskSelectorEvaluator, tgse *tagSelectorEvaluato
 			Tags:           pbv.Tags,
 		}
 		bv.AllowedRequesters = pbv.AllowedRequesters
-		bv.Tasks, bv.EmptyTaskSelectors, errs = evaluateBVTasks(tse, tgse, vse, pbv, tasks)
+		bv.Tasks, emptyTaskSelectors, unmatchedTags, errs = evaluateBVTasks(tse, tgse, vse, pbv, tasks)
+		if len(emptyTaskSelectors) > 0 {
+			bv.TranslationWarnings = append(bv.TranslationWarnings, fmt.Sprintf("buildvariant '%s' has task names/tags that do not match any tasks: '%s'", pbv.Name, strings.Join(emptyTaskSelectors, "', '")))
```

Comment:
What's the difference between a "tag that doesn't match any task" and the unmatched tags below this? I would have thought that an unmatched tag is a subset of empty selectors.

Reply:
I renamed the variables and I redid the errors. I'm not sold on this phrasing either, let me know what you think!